tkp/steps/misc.py

```diff
@@ -137,41 +134,3 @@ def dump_database_backup(db_config, job_dir):
                 db_config['database'], db_config['user'], db_config['password'],
                 output_name
             )
-
-def group_per_timestep(images):
-    """
-    groups a list of TRAP images per time step.
-
-    Per time step the images are order per frequency and then per stokes. The
-    eventual order is:
-
-    (t1, f1, s1), (t1, f1, s2), (t1, f2, s1), (t1, f2, s2), (t2, f1, s1), ...)
-    where:
-
-        * t is time sorted by old to new
-        * f is frequency sorted from low to high
-        * s is stokes, sorted by ID as defined in the database schema
-
-    Args:
-        images (list): list of tuple, (image_db, accessor object)
-
-    Returns:
-        list: List of tuples. The list is sorted by timestamp.
-            Each tuple has the timestamp as a first element,
-            and a list of images sorted by frequency and then stokes
-            as the second element.
-
-    """
-    timestamp_to_images_map = defaultdict(list)
-    for db_image, accessor in images:
-        timestamp_to_images_map[db_image.taustart_ts].append((db_image, accessor))
-
-    # List of (timestamp, [images_at_timestamp]) tuples:
-    grouped_images = timestamp_to_images_map.items()
-
-    # sort the tuples by first element (timestamps)
-    grouped_images.sort()
-
-    # and then sort the nested items per freq and stokes
-    [l[1].sort(key=lambda x: (x[0].freq_eff, x[0].stokes)) for l in grouped_images]
```

Comment:
hm, good point. But is it really a problem? This would only affect the order of the quality check and soure extraction, not source association and other SQL magic. Once it is in the database the order doesn't matter, and that is where the source association happens. Or am I wrong?